autoload/keepeye/system.vim

```diff
@@ -6,8 +6,9 @@ function! keepeye#system#Notify(message)
 endfunction
 
 function! keepeye#system#Bell()
-  if executable('mpv')
-    call system('mpv ' . shellescape(g:keepeye_bell_path) . '&')
+  let l:player =  split(g:keepeye_bell_player, " ")[0]
+  if executable(l:player)
+    call system(g:keepeye_bell_player . ' ' . shellescape(g:keepeye_bell_path) . '&')
```

Comment:
I think the best here is to check first if `g:keepeye_system_bell_cmd` is present. If so, then use it, otherwise loop over the 3 main players, and break one the first one found to play the sound